tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/tuples/BasicTupleTest.java

```diff
@@ -325,6 +328,32 @@ public void testTupleAsTupleFirstMember() {
         BRunUtil.invoke(result, "testTupleAsTupleFirstMember");
     }
 
+    @Test
+    public void testTupleName() {
+        Object returns = BRunUtil.invoke(result, "testTupleName", new Object[]{});
+        TupleValueImpl returnTuple = (TupleValueImpl) returns;
+
+        Type t1 = TypeUtils.getType(returnTuple.get(0));
+        Assert.assertEquals(t1.getName(), "T");
+        Assert.assertEquals(t1.toString(), "[int,string...]");
```

Comment:
If there is a name, why isn't `toString` returning that? Isn't that what we do for records?

Reply:
We're currently computing the string representation and returning, should we change that similar to records for named tuples?